test/func_test/test_acnt.py

```diff
@@ -24,6 +28,53 @@ def supernode_addr(self) -> str:
         """
         return cft.SUPERNODE_ADDR
 
+    def test_pri_str_cons(self, chain: pv.Chain):
+        """
+        test_pri_str_cons tests method from_pri_key_str.
```

Comment:
Can be more descriptive here like "test_pri_str_cons tests constructing Account with private key string."